test/Microsoft.AspNet.Mvc.FunctionalTests/RemoteAttributeValidationTest.cs

```diff
@@ -23,12 +23,12 @@ public RemoteAttributeValidationTest(MvcTestFixture<ValidationWebSite.Startup> f
         public HttpClient Client { get; }
 
         [Theory]
-        [InlineData("Aria", "/Aria")]
+        [InlineData("Area1", "/Area1")]
```

Comment:
The folder names were needed to be consistent in this case because the views now are in their own area and to help view location search. Previously the views were in the root shared folder.